src/server/JasmineGraphInstanceService.cpp

```diff
@@ -1688,7 +1690,7 @@ void *instanceservicesession(void *dummyPt) {
             string trainData(data);
 
             std::vector<std::string> trainargs = Utils::split(trainData, ' ');
-
+            instance_logger.info("Train Data : " + trainData);
```

Comment:
Why do we want the train data to be logged?

Reply:
These are the flags passed for the workers.
Ex:
`--graph_id 1`

I had encountered situations where these data isn't recieved.

I think the `Train data` description is misleading. I'll rename it as `Recieved options: <data>`.
What about it?

Reply:
Yes, that is better message

Reply:
Done